nomos-services/membership/src/backends/sdp.rs

```diff
@@ -0,0 +1,83 @@
+use std::collections::HashMap;
+
+use nomos_sdp::backends::FinalizedBlockEvent;
+use nomos_sdp_core::{DeclarationUpdate, ProviderInfo, ServiceType};
+
+use super::{MembershipBackend, MembershipBackendSettings, MembershipEntry, SnapshotSettings};
+
+pub struct SdpMembershipBackend {
```

Comment:
Could this be called `MockMembershipBackend` and the `Settings` would have Entries from configuration file?

Reply:
The only reason I did not call it MockMembershipBackend but SdpMembershipBackend is because I am getting and event with types from sdp (declarationID, ProviderID, Locators).

But, what you suggested in the PR comment make sense, and mock backend can still use types from sdp, it does not matter, so I will rename it. Thanks!